src/shared/varlink-internal.h

```diff
@@ -0,0 +1,5 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
```

Comment:
Please include `varlink.h`, `fdset.h`, and `stdio.h`, otherwise, this header is not self consistent.